apps/src/gamelab/GameLabSprite.js

```diff
@@ -106,38 +106,24 @@ module.exports.createSprite = function (x, y, width, height) {
   });
 
   // p5.play stores width unscaled, but users in
-  // Game Lab should access and set a scaled version.
-  Object.defineProperty(s, 'width', {
-    enumerable: true,
-    configurable: true,
-    get: function () {
-      if (s._internalWidth === undefined) {
-        return 100;
-      } else {
-        return s._internalWidth * this.scale;
-      }
-    },
-    set: function (value) {
-      s._internalWidth = value / this.scale;
+  // Game Lab should have access to a scaled version.
+  s.getScaledWidth = function () {
+    if (s.width === undefined) {
+      return 100 * this.scale;
+    } else {
+      return s.width * this.scale;
     }
-  });
+  };
```

Comment:
I could be very wrong on this, but I don't think I really needed to use `this.scale` so I changed it to `s.scale` which seems to match what we are doing in the other functions.